lib/winapi/timezoneapi.c

```diff
@@ -0,0 +1,206 @@
+// SPDX-License-Identifier: MIT
+
+// SPDX-FileCopyrightText: 2023 Ryan Wendland
+
+#include <timezoneapi.h>
+#include <winbase.h>
+#include <assert.h>
+#include <xboxkrnl/xboxkrnl.h>
+#include <string.h>
+
+typedef struct {
+    UCHAR Month;     // 1-12
```

Comment:
Can you add a note here that `0` is a valid value representing unknown time zone

Reply:
Note added